impl/pom.xml

```diff
@@ -703,6 +703,12 @@
                             resolution:=optional;cardinality:=multiple,
                             osgi.extender;filter:="(osgi.extender=osgi.serviceloader.processor)";resolution:=optional
                         </Require-Capability>
+                        <Provide-Capability>
+                            osgi.extender;osgi.extender="jsp.taglib";uri="http://java.sun.com/jsf/html";
+                                version:Version="2.3",
+                            osgi.extender;osgi.extender="jsp.taglib";uri="http://java.sun.com/jsf/core";
+                                version:Version="2.3"
+                        </Provide-Capability>
```

Comment:
@stiemannkj1 Just curious -- Did `@spec.version@` not work like it did with https://github.com/javaserverfaces/mojarra/pull/4334 ?

Reply:
I guess they are different files. This one is pom.xml but the other file is bnd.bnd

Reply:
@ngriffin7a, well it wouldn't work as far as I know since this is in the **`pom.xml`**. However, I was just keeping this consistent with the hardcoded version in the `Export-Package` and `Specification-Version`: https://github.com/javaserverfaces/mojarra/pull/4335/files/679280990cddd24dbbb76a407dd83b786bf83af0#diff-a0ca71be2514f2a10cd5ff0a5fa0de7aR654

Probably this should be made into a maven property.